sources/packages/backend/apps/load-test-gateway/src/services/workflow/workflow-data-load.service.ts

```diff
@@ -0,0 +1,126 @@
+import { Injectable } from "@nestjs/common";
+import { InjectRepository } from "@nestjs/typeorm";
+import {
+  Application,
+  EducationProgramOffering,
+  Student,
+  StudentAssessment,
+  StudentAssessmentStatus,
+  User,
+} from "@sims/sims-db";
+import {
+  createFakeApplication,
+  createFakeEducationProgramOffering,
+  createFakeStudent,
+  createFakeStudentAssessment,
+  createFakeUser,
+} from "@sims/test-utils";
+import { DataSource, Repository } from "typeorm";
+import { APPLICATION_DATA_SINGLE_INDEPENDENT } from "../../utils/application.constants";
+
+interface ApplicationPreliminaryData {
+  offering: EducationProgramOffering;
+  student: Student;
+}
+
+@Injectable()
+export class WorkflowDataLoadService {
+  constructor(
+    private readonly dataSource: DataSource,
+    @InjectRepository(Application)
+    private readonly applicationRepo: Repository<Application>,
+    @InjectRepository(EducationProgramOffering)
+    private readonly offeringRepo: Repository<EducationProgramOffering>,
+    @InjectRepository(Student)
+    private readonly studentRepo: Repository<Student>,
+    @InjectRepository(User)
+    private readonly userRepo: Repository<User>,
+    @InjectRepository(StudentAssessment)
+    private readonly assessmentRepo: Repository<StudentAssessment>,
+  ) {}
+
+  /**
+   * Load the application and assessment data required for the load test.
+   * @param dataIterations load test iterations.
+   * @returns student assessments.
+   */
+  async loadApplicationAssessmentData(
```

Comment:
Is the `load` prefix due to the fact that it is loading the data into the DB?
Either way, I would prefer `createSomething` or `saveSomething`?